packages/init/src/steps/set-version.ts

```diff
@@ -1,33 +1,33 @@
 import { CONSTS } from "@slicemachine/core";
 import * as NodeUtils from "@slicemachine/core/build/node-utils";
+import { has } from "fp-ts/lib/Record";
 import { logs } from "../utils";
 
 const defaultSliceMachineVersion = "0.0.41";
 
+function isRecord(o: unknown): o is Record<string, string> {
+  return typeof o === "object" && o !== null && !Array.isArray(o);
+}
+
 const getTheSliceMachineVersionInstalled = (
-  packageJson: NodeUtils.FileContent<NodeUtils.JsonPackage>
+  packageJson: null | NodeUtils.JsonPackage
```

Comment:
really weird that this has to be modified in this PR for the lint :x

Reply:
yeah, it was really annoying i tried a lot things before narrowing the types